sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -3403,6 +3403,11 @@ private void SetNameFieldsIfNull()
                 {
                     return Response.FromValue(false, default);
                 }
+                catch (RequestFailedException storageRequestFailedException)
+                when (storageRequestFailedException.ErrorCode == BlobErrorCode.BlobUsesCustomerSpecifiedEncryption)
```

Comment:
@pakrym @tg-msft This is another example where we'd like to classify error as "info" for the telemetry purposes.
I think centralized place like "response classifier" might not scale to capture corner cases like this. Would it be possible for generated code to accept enumerable of "expected errors" or way to provide local response classifier?